src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -236,6 +238,20 @@ public bool IsSuite
         public ITest Parent { get; set; }
 
         /// <summary>
+        /// Gets the name of the TestWorker that
+        /// is executing this test.
+        /// </summary>
+        public string WorkerId
+        {
+            get {
```

Comment:
We don't use C# 6.0 features. They do not build in all supported platforms, thus the Mono build failure.